### PR TITLE
Interpreter tests

### PR DESCRIPTION
**Pull request by jaanonim**

### CODE REVIEW

File: tests/core/interpreter.test.js [hunk 1]
```diff
@@ -0,0 +1,688 @@
...
+    VarNode
+} = require('../../core/node');
+const Token = require('../../core/token');
+const Position = require('../../core/position');
+const NumberValue = require('../../core/values/number');
+const {
+    RuntimeError
+} = require('../../core/error');
+const Context = require("../../core/context.js");
+0
```
Comment (n2oneProgrammer) [inline tests/core/interpreter.test.js]: ?

File: tests/core/interpreter.test.js [hunk 2]
```diff
@@ -0,0 +1,688 @@
...
+        expect(result.value).toEqual(null);
+        expect(result.error).toBeInstanceOf(RuntimeError);
+    })
+
+    it("Visit EdNode Error", () => {
+        let context = new Context("<global>");
+        context.symbolTable = new SymbolTable();
+
+        let interpreter = new Interpeter(null);
+        let edNode = new EdNode([new VarNode(new Token(Token.TYPE.IDENTIFIER, 'func', null, new Position(0, 0, -1, "fn", "text")))])
```
Comment (n2oneProgrammer) [inline tests/core/interpreter.test.js]: semicolon

File: tests/core/interpreter.test.js [hunk 3]
```diff
@@ -0,0 +1,688 @@
...
+
+        let context = new Context("<global>");
+        context.symbolTable = new SymbolTable();
+        context.symbolTable.set("$who", new WhoValue([0]));
+
+        let mock = jest.fn();
+        testFunc(mock, context);
+
+        let interpreter = new Interpeter();
+        let node = new SaveLoadNode(new Token(Token.TYPE.KEYWORD, "load", null, new Position(-1, 0, -1, "fn", "TO func test END")), new Token(Token.TYPE.PATH, "C:/a.txt", null, new Position(-1, 0, -1, "fn", "TO func test END")))
```
Comment (n2oneProgrammer) [inline tests/core/interpreter.test.js]: semicolon

**Review by n2oneProgrammer — APPROVED** [review]
LGTM and yes i check all lines

**Review by mwawrzen — COMMENTED** [review]
LGTM

**Review by mwawrzen — APPROVED** [review]
LGTMv2

**Pull request merged**